src/main/java/duke/tasklist/TaskList.java

```diff
@@ -0,0 +1,71 @@
+package duke.tasklist;
+
+import duke.tasks.Task;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class represents the list of tasks.
+ * It has operations to add/delete tasks in the list.
+ */
+public class TaskList {
+    private List<Task> list;
+
+    /**
+     * Constructs an empty TaskList.
+     */
+    public TaskList() {
+        this.list = new ArrayList<>();
+    }
+
+    /**
+     * Constructs a TaskList from the given list.
+     * @param list The List to be associated with the TaskList.
+     */
+    public TaskList(List<Task> list) {
+        this.list = list;
+    }
+
+    public Task get(int i) {
```

Comment:
Maybe a clearer naming would be `getTask(int index)`?

Reply:
Oops I actually feel like a short name seems appropriate since this var is only used once in this one-liner fxn >< But I see your point here :D Perhaps adding javadocs to describe the param `i` might help instead? Just my two cents 😬